### PR TITLE
tests_l2: Update qatlib workload to ubi9 rpm packages

### PR DESCRIPTION
**Pull request by vbedida79**
- Updated qatlib buildconfig to ubi9 with rpm package instead of building from source
- Updated buildconfig, readme to use redhat account credentials for subscription-manager to install rpm packages from specific repos
- qatlib job command updated
- enhancements of entitlement builds and secret build inputs will be added in further releases

Signed-off-by: vbedida79 [veenadhari.bedida@intel.com](mailto:veenadhari.bedida@intel.com)

### CODE REVIEW

File: tests/l2/README.md [hunk 1]
```diff
@@ -113,10 +113,12 @@ $ oc logs intel-dgpu-hwinfo-l4572
 ```                        
 
 ### Verify Intel® QuickAssist Technology provisioning
-This workload runs [qatlib](https://github.com/intel/qatlib) sample tests. Refer to the [qatlib readme](https://github.com/intel/qatlib/blob/main/INSTALL) for more details. 
+This workload runs [qatlib](https://github.com/intel/qatlib) sample tests using RPM packages. Refer to the [qatlib readme](https://github.com/intel/qatlib/blob/main/INSTALL) for more details. 
 
 *	Build the workload container image
 
+Please replace the credentials in buildconfig yaml with your RedHat account login credentials. Updates with entitlement builds and enhancements for credentials as secret/build inputs will be added in the next releases.
```
Comment (uMartinXu) [inline tests/l2/README.md:120]: Can you add the link about entitlement builds and enhancements for credential as secret/build? Or
Do you think we can just remove the second sentence about entitlement builds and enhance ments?

File: tests/l2/README.md [hunk 2]
```diff
@@ -113,10 +113,12 @@ $ oc logs intel-dgpu-hwinfo-l4572
 ```                        
 
 ### Verify Intel® QuickAssist Technology provisioning
-This workload runs [qatlib](https://github.com/intel/qatlib) sample tests. Refer to the [qatlib readme](https://github.com/intel/qatlib/blob/main/INSTALL) for more details. 
+This workload runs [qatlib](https://github.com/intel/qatlib) sample tests using RPM packages. Refer to the [qatlib readme](https://github.com/intel/qatlib/blob/main/INSTALL) for more details.
```
Comment (uMartinXu) [inline tests/l2/README.md:116]: using RedHat building and distributing QAT RPM package from repo codeready-builder-for-rhel-9-x86_64-rpms. (Correct me if the repo name is not accurate. )

File: tests/l2/README.md (hunk 1 above)
Comment (vbedida79) [reply tests/l2/README.md:120]: Sure, we can remove it. entitlement builds has not worked as of yet. enhancement to pass credentials as secret is possible.

Comment (vbedida79) [reply tests/l2/README.md]: updated

File: tests/l2/README.md (hunk 2 above)
Comment (vbedida79) [reply tests/l2/README.md]: updated

**Comment by uMartinXu** [comment]
LGTM

**Pull request merged**